ts/Series/DependencyWheel/DependencyWheelSeriesDefaults.ts

```diff
@@ -104,6 +104,18 @@ const DependencyWheelSeriesDefaults: DependencyWheelSeriesOptions = {
 
     curveFactor: 0.6,
 
+    /**
+     * The corner radius of the border surrounding each node. A number
+     * signifies pixels. A percentage string, like for example `50%`, signifies
+     * a relative size. For nodes this is relative to the node width.
+     *
+     * @type    {number|string|Highcharts.BorderRadiusOptionsObject}
+     * @default 3
+     * @product highcharts
+     * @since   11.0.0
+    */
+    borderRadius: 3,
```

Comment:
Do we need to actually set it here? It seems to me that the default is already 3, probably it gets inherited from the sankey series? In that case we can just leave the doclet without actually changing the working code.

Reply:
Thank you for your suggestion. That makes sense.